museum_export/handler.py

```diff
@@ -0,0 +1,68 @@
+# handler.py
+""" Module to launch application """
+
+import os
+import sys
+import json
+from pathlib import Path
+where_i_am = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(where_i_am)
+sys.path.append(where_i_am + "/dependencies")
+from dependencies.pipelineutilities.pipeline_config import get_pipeline_config, load_config_ssm  # noqa: E402
+import dependencies.sentry_sdk as sentry_sdk  # noqa: E402
+from dependencies.sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration  # noqa: E402
+from process_web_kiosk_json_metadata import processWebKioskJsonMetadata  # noqa: E402
+from dependencies.pipelineutilities.google_utilities import establish_connection_with_google_api, execute_google_query, build_google_query_string  # noqa: #402
+
+
+def run(event, context):
+    """ run the process to retrieve and process web kiosk metadata """
+    _suplement_event(event)
+    config = get_pipeline_config(event)
+    config = load_config_ssm(config['google_keys_ssm_base'], config)
+    config = load_config_ssm(config['museum_keys_ssm_base'], config)
+    # print("config-process-bucket=", config['process-bucket'])
+    # config['process-bucket'] = 'new-csv-processbucket-10dr776tnq9be'
+
+    if config != {}:
```

Comment:
empty dictionaries evaluate to False in python.  It is probably better to do just if config  or if bool(config)